contents/handbook/engineering/ee-setup.md

```diff
@@ -45,3 +51,12 @@ Currently for internal use only.
   - Tests:
       - All the same, except skip `DEBUG=1` in the env settings.
       - Set as the "target" in run configuration `ee/clickhouse`
+
+### Post Development Cleanup
+
+Awesome! You've made your awesome fix/feature/contribution and you're ready to call it day. Make sure to `docker-compose down` your app and manually clean up your Clickhouse database:
+
+```
+docker compose -f ee/docker-compose.ch.yml down
+DEBUG=1;DJANGO_SETTINGS_MODULE=posthog.settings;PRIMARY_DB=clickhouse;CLICKHOUSE_HOST=clickhouse;CLICKHOUSE_DATABASE=posthog;CLICKHOUSE_SECURE=false;CLICKHOUSE_VERIFY=false python migrate.py migrate_clickhouse
```

Comment:
Perhaps this should then be a npm script as well?

Reply:
With the semi colons, this implies it to be separate commands, I don't think those are set as env variables then?

I might be mistaken, but last I tried this, I had to replace the semi colons with spaces so that this becomes a single command and python got those env vars.

Reply:
This might work depending on your shell, e.g. if you run `set -a` before and it'll all work. However @neilkakkar 's point is legit -> bash normally doesn't export variables. This means `python` won't see them unless you use `export DEBUG=1; export ...`.

Adding the envs with spaces before the actual command helps with exporting, but the real question is: do we need all of these? E.g. when is the django settings module something else?

Reply:
Tested this out again and it looks like the cli command itself is wrong! It should be `python manage.py migrate_clickhouse` not `python migrate.py migrate_clickhouse`.

On second thought, I'm not even sure if this manual cleanup step is required, since we're removing the entire clickhouse container on `compose down`. I actually haven't had any issues with leftover clickhouse data w just running compose down so I'm considering deleting that second command altogether. Then I don't think another npm script is warranted, that's just proper Docker etiquette 😄